Vagrantfile

```diff
@@ -187,3 +214,50 @@ Vagrant.configure("2") do |config|
     end
   end
 end
+
+
+# Get .onion URL for connecting to instances over Tor.
+# The Ansible playbooks fetch these values back to the
+# Admin Workstation (localhost) so they can be manually
+# added to the inventory file. Possible values for filename
+# are "app-ssh-aths" and "mon-ssh-aths".
+def find_ssh_aths(filename)
+  require 'find'
+  repo_root = File.expand_path(File.dirname(__FILE__))
+  ansible_dir = File.join(repo_root, "install_files", "ansible-base")
+
+  Find.find(ansible_dir) do |path|
```

Comment:
A recursive search of the subdirectories under `ansible_dir` seems unnecessary, since `*-ssh-aths` are always placed in `ansible_dir`. Can't you just get the file from the well-known path?